manual.md

```diff
@@ -52,31 +52,31 @@ Before a RGP can be submitted on-chain, it must first complete **an off-chain r
 
 **Formal Review**
 
 The _Formal Review_ phase serves as the final review step in the governance process before proposals are submitted on-chain. Proposals in this phase incorporate feedback from the _Discussion_ phase, and are accompanied by an off-chain Snapshot poll to gauge consensus for their advancement. All Snapshot polls must begin on **5:00pm GMT+2** the **third Monday of the month** and end at **5:00pm GMT+2** on the **following Monday**. Each Snapshot poll should have three voting options: `Yes` `No` and `Abstain`. 
 
 During _Formal Review_, proposal authors are required to:
-- Publish an new version of their proposal on the forum that incorporates feedback from the *Discussion* phase. Proposals in this stage should be posted under the [Active Proposals](https://community.radworks.org/c/governance/proposals/18) category on the forum. The phase in the title should read [Formal Review].
-- If applicable, inlcude a draft of the executable proposal code for the community to review. 
-- Publish a corresponding Snapshot poll to your proposal. To initiate a Snapshot poll for Formal Review, you must be added as an 'author' in our Snapshot space. This measure is in place to prevent spam proposals. Please reach out to Shelby (shelb_ee#9785 on Discord) with the Ethereum address of the party posting your proposal.
-- At the top of the Snapshot poll, please include links to both the _Discussion_ and _Formal Review_ forum posts.
-- Once live, proposal authors should include a link to the Snapshot poll as a callout at the top of their proposal draft on the forum, and clearly state when the poll closes.
+- Publish an new version of their proposal on the forum that incorporates feedback from the *Discussion* phase. Please use [this template]([url](https://github.com/radicle-foundation/radworks-governance/blob/main/templates/05-Formal-Review-forum-post-header)) as guide. Proposals in this stage should be posted under the [Active Proposals](https://community.radworks.org/c/governance/proposals/18) category on the forum. The phase in the title should read [Formal Review].
+- If applicable, include a draft of the executable proposal code for the community to review in the [proposal code folder]([url](https://github.com/radicle-foundation/radworks-governance/tree/main/proposals)). Use [this template]([url](https://github.com/radicle-foundation/radworks-governance/blob/main/templates/07-proposal-code-template.md)) to draft the proposal code. 
+- Publish a corresponding Snapshot poll to your proposal on the [Radworks Snapshot page]([url](https://snapshot.org/#/gov.radworks.eth)). To publish a Snapshot poll for Formal Review, you must be added as an 'author' in the Radworks Snapshot space. This measure is in place to prevent spam proposals. Please reach out to Shelby (shelb_ee#9785 on Discord) with the Ethereum address of the party posting your proposal.
```

Comment:
Should we say: If you need the Governance Committee can submit the Snapshot poll for you, please reach out to Shelby before 3pm on Monday. or something?